.github/CONTRIBUTING.md

```diff
@@ -116,3 +116,9 @@ The package maintainer also reserves the rights to manually adjust the criteria
 ## Any questions
 
 If you have any further questions regarding contribution - please reach out to the repository maintainer!
+
+<!-- urls -->
+[url_pkgdown]: https://insightsengineering.github.io/r.pkg.template/index.html
+[url_getting_started]: https://insightsengineering.github.io/r.pkg.template/articles/r-pkg-template.html
```

Comment:
please note the change of dots into hyphens, e.g. https://insightsengineering.github.io/teal.logger/articles/teal-logger.html

Reply:
Good catch.